solidity/contracts/Peggy.sol

```diff
@@ -36,6 +37,15 @@ contract Peggy {
 		uint256 _amount,
 		uint256 _eventNonce
 	);
+	event ERC20DeployedEvent(
+		// FYI: Can't index on a string without doing a bunch of weird stuff
+		string _cosmosDenom,
+		address _tokenContract,
+		string _name,
+		string _symbol,
+		uint8 _decimals,
+		uint256 _eventNonce
```

Comment:
we should index on the event nonce and tokencontract

Reply:
Why are we indexing on any of this stuff?

Reply:
we don't use it but someone might. This is good now.